sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_models.py

```diff
@@ -15,18 +14,18 @@
     Glossary as _Glossary
 )
 
+
 class TranslationGlossary(object):  # pylint: disable=useless-object-inheritance
-    """Glossary / translation memory for the request.
-
-    :param glossary_url: Required. Location of the glossary.
-     We will use the file extension to extract the formatting if the format parameter is not
-     supplied.
-     If the translation language pair is not present in the glossary, it will not be applied.
-    :type glossary_url: str
-    :param str file_format: Format.
-    :keyword str format_version: Format version.
-    :keyword storage_source: Storage Source. Default value: "AzureBlob".
-    :paramtype storage_source: str
+    """Glossary / translation memory to apply to the translation.
+
+    :param str glossary_url: Required. Location of the glossary file. This should be a SAS URL to
+        the glossary file in the storage blob container. If the translation language pair is
+        not present in the glossary, it will not be applied.
+    :param str file_format: Required. Format of the glossary file. To see supported formats,
+        call the :func:`~DocumentTranslationClient.get_glossary_formats()` client method.
+    :keyword str format_version: File format version.
```

Comment:
Consider adding that if not passed, the service will use the File format `DefaultVersion`